src/zoid/buttons/util.js

```diff
@@ -121,6 +121,49 @@ export function getVenmoExperiment() : VenmoExperiment {
     }
 }
 
+export function createNoPaylaterExperiment() : Experiment | void {
+    const disableFunding = getDisableFunding();
+    const isDisableFundingPaylater = disableFunding && disableFunding.indexOf(FUNDING.PAYLATER) !== -1;
+    const enableFunding = getEnableFunding();
+    const isEnableFundingPaylater = enableFunding && enableFunding.indexOf(FUNDING.PAYLATER) !== -1;
+
+    const { paylater } = getFundingEligibility();
+    const isEligibleForPaylater = paylater?.eligible;
+    const isExperimentable = paylater?.products?.paylater?.variant === 'experimentable' || paylater?.products?.payIn4?.variant === 'experimentable';
+    // No experiment because ineligible, already forced on or off
+    if (!isEligibleForPaylater
+        || !isExperimentable
+        || isDisableFundingPaylater
+        || isEnableFundingPaylater
+    ) {
+        return;
+    }
+
+    if (isDevice()) {
```

Comment:
Do we need to have all of these types of disables by platform?  Or can we just have one for all?

Reply:
Not sure yet, confirming with analytics, but since the logs I gave them followed this pattern I was planning on leaving it unless they confirm the generalized form